typescript/packages/subsurface-viewer/src/storybook/layers/Axes2DLayer.stories.tsx

```diff
@@ -49,6 +49,33 @@ export const Base: StoryObj<typeof SubsurfaceViewer> = {
     },
 };
 
+function MakeLabel(a: number): string {
+    // Choos exponential format with 3 digits after point.
+    const label = d3.format(".3e")(a);
+    //const label = d3.format(".1f")(a) // fixed decimal  KEEP
```

Comment:
Can we remove the comment?

Reply:
I thought I would keep this one as otherwise one has to look up d3 documentation to test dofferent cases,